_docs/_latest/configure/scanner/rules.md

```diff
@@ -474,6 +474,26 @@ rules:
   * **Valid values**: String.
   * **Example values**: `bigquery-json.googleapis.com`, `logging.googleapis.com`
 
+
+## External Project Access rules
+
+### Rule definitions
+```yaml
+rules:
+- name: Only allow access to projects in my organization.
+  allowed_ancestors:
+  - organizations/{ORGANIZATION_ID}
+```
+
+* `name`
+  * **Description**: The name of the rule.
+  * **Valid values**: String.
+  
+* `allowed_ancestors`
+  * **Description**: The folder or parent that is allowed as an ancestor of a project.  This must follow the pattern of valid values
```

Comment:
Does this really support folder for this release?

Reply:
as per offline discussion, we do support folder